### PR TITLE
add example manual verification of quote signature with ecc

### PR DESCRIPTION
**Pull request by JacobBarthelmeh**
`./examples/pcr/quote -ecc`

### CODE REVIEW

**Review by dgarske — CHANGES_REQUESTED** [review]
Is it possible you could add support for this with RSA also?

File: examples/pcr/quote.c [hunk 1]
```diff
@@ -229,6 +266,60 @@ int TPM2_PCR_Quote_Test(void* userCtx, int argc, char *argv[])
...
+
+        /* try to confirm the signature */
+        if (rc == 0) {
+            rc = wc_ecc_verify_hash_ex(&r, &s, digest, WC_SHA256_DIGEST_SIZE,
+                &res, &ecKey);
+        }
+        mp_free(&r);
+        mp_free(&s);
+        wc_ecc_free(&ecKey);
+        printf("%s [rc = %d, result = %d]\n", (res == 1)? "SUCCESS": "FAILURE",
```
[the remark this answers is not in the document]
Comment (JacobBarthelmeh) [reply examples/pcr/quote.c:320]: Possibly different for swtpm used? Try with ms-tpm-20-ref. I made a commit now updating the docs/SWTPM.md with steps for a test done.

**Comment by JacobBarthelmeh** [comment]
Yup, can add in RSA. Would like to follow up with it though in a separate PR, keeping this one with as few code changes as can for the use.

**Review by dgarske — APPROVED** [review]
Works for me now! Thank you

**Pull request merged**